code/modules/holiday/holidays.dm

```diff
@@ -67,10 +70,11 @@
 
 /datum/holiday/valentines
 	name = VALENTINES
-	begin_day = 9 //6 days early
+	begin_day = 14
```

Comment:
You don't. The event lasts two days to make up for the fact that it starts earlier in some timezones than in others. It already works like this, a lot of events (Christmas for example) start one day or a few days early so everybody gets to experience them when they're happening in their country.